src/idsk/components/footer-extended/footer-extended.js

```diff
@@ -0,0 +1,241 @@
+  }
+
+
+/**
+ * Footer for extended websites
+ */
+class FooterExtended {
+    constructor($module) {
+        this.$module = $module;
+    }
+
+    init() {
+        console.log('test');
+
+        let $module = this.$module;
+        // check for module
+        if (!$module) {
+            return;
+        }
+
+
+        let $yesButton = $module.querySelector('#idsk-yes-button');
+        let $noButton = $module.querySelector('#idsk-no-button');
+        let $errorButton = $module.querySelector('#idsk-error-button');
+        let $closeErrorFormButton = $module.querySelector('#close-error-form-button');
+        let $closeHelpFormButton = $module.querySelector('#close-help-form-button');
+
+        let $writeUsButton = this.$module.querySelector('#idsk-write-us-button');
+
+        if ($yesButton && $noButton && $errorButton) {
+            $yesButton.addEventListener('click', this.handleYesButtonClick.bind(this));
+            $noButton.addEventListener('click', this.handleNoButtonClick.bind(this));
+            $errorButton.addEventListener('click', this.handleErrorButtonClick.bind(this));
+        }
+
+        if ($writeUsButton) {
+            $writeUsButton.addEventListener('click', this.handleErrorButtonClick.bind(this));
+        }
+
+        if ($closeHelpFormButton) {
+            $closeHelpFormButton.addEventListener('click', this.handleCloseHelpFormButtonClick.bind(this));
+        }
+
+        if ($closeErrorFormButton) {
+            $closeErrorFormButton.addEventListener('click', this.handleCloseErrorFormButtonClick.bind(this));
+        }
+
+    }
+
+
+    hideAllElements() {
+        let $yesOption = this.$module.querySelector('#idsk-yes-option');
+        let $noOption = this.$module.querySelector('#idsk-help-form');
+        let $errorOption = this.$module.querySelector('#idsk-error-form');
+
+        this.addClass($yesOption, 'idsk-display-none');
+        this.addClass($noOption, 'idsk-display-none');
+        this.addClass($errorOption, 'idsk-display-none');
+    }
+
+    handleYesButtonClick(e) {
+        //this.hideAllElements();
+
+        let $feedbackQuestion = this.$module.querySelector('#feedback-question');
+
+        let $yesOption = this.$module.querySelector('#idsk-yes-option');
+        let $noOption = this.$module.querySelector('#idsk-help-form');
+        let $errorOption = this.$module.querySelector('#idsk-error-form');
+
+
+
+        let $infoQuestion = this.$module.querySelector('#idsk-info-question');
+        let $heartSymbol = this.$module.querySelector('#idsk-heart');
+
+        //this.addClass($yesOption, 'idsk-display-none');
+        this.addClass($noOption, 'idsk-display-none');
+        this.addClass($errorOption, 'idsk-display-none');
+
+
+        this.toggleClass($infoQuestion, 'idsk-heart');
+        // this.toggleClass($yesButton, 'idsk-display-none');
+        // this.toggleClass($noButton, 'idsk-display-none');
+        // this.toggleClass($errorButton, 'idsk-display-none');
+
```

Comment:
zakomentovany kod povymazuj, prazdne riadky teiz, zbytocne dlhe to je, zle sa to potom cita, obcas nejaky komentar aspon nad metofdy som dal ze co robia v skratke